### PR TITLE
BHBC-781: Database model updates

### PR DESCRIPTION
**Pull request by NickPhura**
# Overview

## Links to jira tickets

Only a subset of https://quartech.atlassian.net/browse/BHBC-781

## This PR contains the following changes

Add new migration to include the most recent db model changes from Charlie, relating to the project funding data.

Moved the contents of one of the seeds into the migration it was originally a part of. This aligns better with what seeding should be used for.
- The seeds should only contain ephemeral information that is needed locally to make the app work and/or to pre-populate it with some test data so that it doesn't start up completely empty, and not core persistent data like code table values.

## This PR contains the following types of changes

- [x] New feature (change which adds functionality)
- [ ] Enhancement (improvements to existing functionality)
- [ ] Bug fix (change which fixes an issue)
- [x] Misc cleanup / Refactoring / Documentation
- [ ] Version change

## Checklist

A list of items that are good to consider when making any changes.

_Note: this list is not exhaustive, and not all items are always applicable._

### General

- [x] I have performed a self-review of my own code

### Code

- [ ] New files/classes/functions have appropriately descriptive names and comment blocks to describe their use/behaviour
- [ ] I have avoided duplicating code when possible, moving re-usable pieces into functions
- [ ] I have avoided hard-coding values where possible and moved any re-usable constants to a constants file
- [ ] My code is as flat as possible (avoids deeply nested if/else blocks, promise chains, etc)
- [ ] My code changes account for null/undefined values and handle errors appropriately
- [ ] My code uses types/interfaces to help describe values/parameters/etc, help ensure type safety, and improve readability

### Style

- [ ] My code follows the established style conventions
- [ ] My code uses native material-ui components/icons/conventions when possible

### Documentation

- [ ] I have commented my code sufficiently, such that an unfamiliar developer could understand my code
- [ ] I have added/updated README's and related documentation, as needed

### Tests

- [ ] I have added/updated unit tests for any code I've added/updated
- [ ] I have added/updated the Postman requests/tests to account for any API endpoints I've added/updated

### Linting/Formatting

- [ ] I have run the linter and fixed any issues, as needed  
       _See the `lint` commands in package.json_
- [ ] I have run the formatter and fixed any issues, as needed  
       _See the `format` commands in package.json_

### SonarCloud

- [ ] I have addressed all SonarCloud Bugs, Vulnerabilities, Security Hotspots, and Code Smells

## How Has This Been Tested?

Locally in docker.

### CODE REVIEW

**Comment by NickPhura** [comment]
As part of updating that past migration (to better align the usage of seeds), I will need to blow away dev before this gets merged (or after, but more convenient to do it first).  So if you review this and approve, don't merge it for me.

**Review by sdevalapurkar — APPROVED** [review]
Looks good!! We looked at a lot of this when you were walking me through some stuff yesterday so it all seems right to me :) will hold off on merge

**Pull request merged**
